ledger-api/README.md

```diff
@@ -2,15 +2,11 @@
 
 This is the API code for the ledger, which contains:
  * gRPC API definitions
- * Generated Scala bindings
+ * Generated Scala bindings and client
```

Comment:
I am probably being daft , but what is the reason for this change ? thought we are removing all mentions of it

Reply:
This is a README of an internal directory which lists the directory's content.